code/modules/shuttle/supply.dm

```diff
@@ -1,10 +1,17 @@
 #define MAX_CRATE_DELIVERY 40
 
+/// This can be shipped on the shuttle.
 #define CARGO_OK 0
+/// This can never be shipped on the shuttle.
 #define CARGO_PREVENT_SHUTTLE 1
+/// Ignore this atom.
 #define CARGO_SKIP_ATOM 2
-#define CARGO_REQUIRE_PRIORITY 3
-#define CARGO_HAS_PRIORITY 4
+/// Must contain a priority or wrong item.
+/// Used to prevent opening a locked box by selling the box and getting the
+/// contents back.
```

Comment:
```suggestion
/// Must contain a priority or wrong item. Used to prevent opening a locked box by selling the box and getting the contents back.
```
Won't be fully autodoc'd without this